src/app/components/BibPage/BibDetails.jsx

```diff
@@ -6,234 +6,78 @@ import {
-      bibValues,
-      identifierType,
-    );
-    if (Array.isArray(entities) && entities.length > 0) {
-      const markup = entities.map((ent) => {
-        const nodes = [<span key={`${ent['@value']}`}>{ent['@value']}</span>];
-        if (ent.identifierStatus)
-          nodes.push(
-            <span key={`${ent['@value']}`}>
-              {' '}
-              <em>({ent.identifierStatus})</em>
-            </span>,
-          );
-        return nodes;
-      });
-
-      return markup.length === 1 ? (
-        markup.pop()
-      ) : (
-        <ul>
-          {markup.map((mark) => (
-            <li key={mark[0].key}>{mark}</li>
-          ))}
-        </ul>
-      );
-    }
-    return null;
-  }
-
-  /**
-   * getDefinition(bibValues, fieldValue, fieldLinkable, fieldIdentifier,
-   * fieldSelfLinkable, fieldLabel)
+   * getDefinition(bibValues, fieldValue, fieldLinkable,
+   *   fieldSelfLinkable, fieldLabel)
    * Gets a list, or one value, of data to display for a field from the API.
    *
    * @param {array} bibValues - the value(s) of the current field
    * @param {string} fieldValue - the name of the current field
    * @param {boolean} fieldLinkable  - flags true if the field should be clickable
-   * @param {string} fieldIdentifier
    * @param {string} fieldSelfLinkable - flags true if the Bib field already has a URL
    * @param {string} fieldLabel - offers the type of search keywords
    * @return {HTML element}
    */
-  getDefinition(
+  const getDefinition = (
     bibValues,
     fieldValue,
     fieldLinkable,
-    fieldIdentifier,
     fieldSelfLinkable,
     fieldLabel,
-  ) {
-    if (fieldValue === 'identifier') {
-      return this.getIdentifiers(bibValues, fieldIdentifier);
-    }
-
+  ) => {
+    // If there is only one value object in the array for the bib,
+    // then render this definition value as a simple `span`, react-router
+    // `link`, or an `a` tag.
     if (bibValues.length === 1) {
       const bibValue = bibValues[0];
-
       const url = `filters[${fieldValue}]=${bibValue}`;
-      return this.getDefinitionOneItem(
+      return renderSingleValue(
         bibValue,
         url,
-        bibValues,
         fieldValue,
         fieldLinkable,
-        fieldIdentifier,
         fieldSelfLinkable,
         fieldLabel,
       );
     }
 
+    // Otherwise, render every value object in the array for the bib in a list.
+    // Each list item definition will be rendered as a simple `span`,
+    // react-router `link`, or an `a` tag.
     return (
       <ul>
-        {bibValues.map((value) => {
-          const queryString =  typeof value === 'string' ? value : value.label;
-          const url = `filters[${fieldValue}]=${encodeURIComponent(queryString)}`
+        {bibValues.map((value, index) => {
+          const queryString = typeof value === 'string' ? value : value.label;
+          // @NOTE this encodes ">" as "%3E" so in "renderSingleValue",
```

Comment:
This `@NOTE` doesn't always seem to happen in production. I think it's because the data is different. But one QA example is https://qa-www.nypl.org/research/research-catalog/bib/b22092977 where most of the subjects toward the end of the list have "undefined" in their URL.